core/src/main/scala/cats/syntax/parallel.scala

```diff
@@ -40,6 +54,12 @@ final class ParallelSequenceOps[T[_], M[_], A](val tma: T[M[A]]) extends AnyVal
     Parallel.parSequence(tma)
 }
 
+final class ParallelSequence_Ops[T[_], M[_], A](val tma: T[M[A]]) extends AnyVal {
```

Comment:
Totally nitpicking, the `_` in the class name is a bit unconventional in this code base. Normally it wouldn't be a big deal, we could always change it after, but we can't really in cats....

Reply:
I kind of agree, but couldn't come up with a better name. Any suggestions?